### PR TITLE
Add Trino birthday banner

### PR DESCRIPTION
**Pull request by cindyngo44**
Added the new birthday banner that will link to the Birthday Blog post
image

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -21,6 +21,15 @@ title: Distributed SQL query engine for big data
   </div>
 </div>
 
+<div class="trino-birthday-banner">
+  <div class="trino-birthday-content">
+    <img src="/assets/images/trino-summit-logo.png" class="trino-birthday-logo" />
```
Comment (mosabua) [inline index.md:26]: this needs to be changed to birthday image..

Comment (cindyngo44) [reply index.md:26]: Currently waiting on those

**Comment by cindyngo44** [comment]
Added the new birthday banner that will link to the Birthday Blog post
image

File: index.md [hunk 2]
```diff
@@ -21,6 +21,14 @@ title: Distributed SQL query engine for big data
   </div>
 </div>
 
+<div class="trino-birthday-banner">
+  <div class="trino-birthday-content">
+    <h1>Trino is celebrating its 10th Birthday!</h1>
+    <a class="btn trino-birthday-button" href="./blog/2022/08/08/trino-tenth-birthday.html">Click to learn more</a>
```
Comment (mosabua) [inline index.md:27]: Just use 

Learn more


it should be obvious that it is a button

Comment (cindyngo44) [reply index.md]: updated!

File: index.md [hunk 3]
```diff
@@ -21,6 +21,14 @@ title: Distributed SQL query engine for big data
   </div>
 </div>
 
+<div class="trino-birthday-banner">
+  <div class="trino-birthday-content">
+    <h1>Trino is celebrating its 10th Birthday!</h1>
+    <a class="btn trino-birthday-button" href="./blog/2022/08/08/trino-tenth-birthday.html">Learn more</a>
```
Comment (mosabua) [inline index.md:27]: use the post_url syntax please

File: index.md [hunk 4]
```diff
@@ -21,6 +21,14 @@ title: Distributed SQL query engine for big data
...
+    <img src="/assets/images/trino-birthday-banner-peek.png" class="trino-birthday-peek">
```
Comment (mosabua) [inline index.md:28]:
```suggestion
    <img src="./assets/images/trino-birthday-banner-peek.png" class="trino-birthday-peek">
```

**Review by mosabua — APPROVED** [review]
Great.. ready to merge on Monday now! Thank you.

**Pull request merged**